### PR TITLE
document liveBackup() usage

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

**Comment by ralphtheninja** [comment]
@juliangruber Merge if you're happy with `liveBackup()` docs.

File: README.md [hunk 1]
```diff
@@ -35,6 +41,19 @@ db.put('name', 'Level', function (err) {
 })
 ```
 
+The `liveBackup()` is accessibly on the underlying `LevelDOWN-Hyper` object:
```
Comment (juliangruber) [inline README.md]: *accessible, otherwise +1!

Comment (ralphtheninja) [inline README.md]: Good catch!

**Comment by ralphtheninja** [comment]
1.2.1

**Pull request merged**